src/Services/validators.ts

```diff
@@ -1,3 +1,5 @@
+import { Connection } from "promise-mysql";
```

Comment:
```suggestion
import type { Connection } from "promise-mysql";
```

I think you only want the `type`, so this is how you import only the type (stops this import being put into the JavaScript, so it helps us without wasting time in the built app)

Reply:
done